### PR TITLE
Use startsWith and endsWith from es6-shim

### PR DESCRIPTION
**Pull request by floatdrop**
Since upcoming releases of node will support this methods we could use them now with shim and remove shim later.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "main": "./api.js",
   "author": "",
   "dependencies": {
+    "es6-shim": "^0.27.1",
```
Comment (guybedford) [inline package.json]: Can we rather use core-js here do you think? Since it is already in the dependency chain through Babel.

File: api.js [hunk 2]
```diff
@@ -14,6 +14,8 @@
  *   limitations under the License.
  */
 
+require('es6-shim');
```
Comment (guybedford) [inline api.js]: This probably needs to be in https://github.com/jspm/jspm-cli/blob/master/cli.js as well? Or every file?

**Comment by guybedford** [comment]
Sure, happy to go this route.

File: package.json (hunk 1 above)
Comment (floatdrop) [inline package.json]: Didn't know about it, looks nicer with partial require.

File: api.js (hunk 2 above)
Comment (floatdrop) [inline api.js]: My bad. I should have put them in every file, that needs shims.

**Comment by floatdrop** [comment]
Rebased and squashed.

**Comment by guybedford** [comment]
Awesome, thanks!

**Pull request merged**